.github/workflows/tests.yml

```diff
@@ -295,9 +295,9 @@ jobs:
       if: failure()
```

Comment:
Please take this file out of the PR.
Changes to the CI are independent of pyramids.